.github/auto_assign.yml

```diff
@@ -0,0 +1,13 @@
+addReviewers: true
+numberOfReviewers: 0
+reviewers:
+  - vuhuucuong
+  - zlatanpham
+  - 4nmt
+  - danndz
+  - NguyenDiem
+  - putrann
+  - trongnguyen2706
+  - vdhieu
+skipKeywords:
+  - wip
```

Comment:
How exactly does it work? Can we skip the job if the PR is in `draft`?

Reply:
If the PR's title includes `wip` keyword (case insensitive),  then this action will be skipped
Draft PRs are auto skipped as I saw here https://github.com/kentaro-m/auto-assign-action/blob/master/src/handler.ts#L47-L52